libraries/tock-register-interface/src/registers.rs

```diff
@@ -33,7 +33,15 @@ use crate::{RegisterLongName, UIntLike};
 /// [`ReadWriteable`](crate::interfaces::ReadWriteable) traits are
 /// implemented.
 // To successfully alias this structure onto hardware registers in memory, this
-// struct must be exactly the size of the `T`.
+// struct must be exactly the size of the `T` and is thus marked
+// `repr(transparent)` over an `UnsafeCell<T>`, which itself is
+// `repr(transparent)` over `T`.
+//
+// This struct is constructed by casting a pointer to it (or, implicitly, by
+// casting a pointer to a larger struct that containts this type). As such, it
+// does not have a public constructor and Rust thinks its dead code and should
```

Comment:
```suggestion
// does not have a public constructor and Rust thinks it's dead code and should
```

Reply:
Hah, I thought I copy-pasted this :smile: